timewarp_gl/plugin.cpp

```diff
@@ -223,6 +254,45 @@ class timewarp_gl : public threadloop {
 		return pixels;
 	}
 
+	// this is the inverse of what they do in Monado
+	// is there a better way to handle this?
+	GLuint ConvertVkFormatToGL(uint64_t vk_format, GLint swizzle_mask[]) {
+		switch (vk_format) {
+			case VK_FORMAT_R8G8B8A8_UNORM: return GL_RGBA8;
+			case VK_FORMAT_B8G8R8A8_SRGB: {
+				swizzle_mask[0] = GL_BLUE;
+				swizzle_mask[2] = GL_RED;
+			}
+			case VK_FORMAT_R8G8B8A8_SRGB:  return GL_SRGB8_ALPHA8;
+			default: return 0;
+		}
+	}
+
+	void VulkanGLInterop(const vk_image_handle& vk_handle, int swapchain_index) {
```

Comment:
Nicely done!

One minor comment is that both `glGenTextures()` and `glCreateMemoryObjectsEXT()` can create multiple handles in a single call, so we can potentially reduce the number of GL calls if we refactor the code. But this is super minor, and mostly just a general tip.